buildenv/docker/mkdocker.sh

```diff
@@ -187,6 +199,15 @@ validate_options() {
         ;;
     esac
   fi
+
+  if [ $jdks_to_build = all ] ; then
+    bootjdks="8 11 16"
+    extensionrepos="8 11 16 17"
+  else
+    local jdk_versions="${jdks_to_build//,/ }"
+    bootjdks="${jdk_versions/17/16}"
+    extensionrepos=$jdk_versions
+  fi
```

Comment:
JDK 16 should not be in this list (it won't build anymore).

Quotes are needed here in case $jdk_versions contains spaces.
`bootjdk_versions` needs to be derived from `jdk_versions`:
```
  if [ $jdk_versions = all ] ; then
    jdk_versions="8 11 17 next"
  fi
  bootjdk_versions="${jdk_versions/17/16}"
  bootjdk_versions="${bootjdk_versions/next/17}"
```